### PR TITLE
krita: add page

### LINKED ISSUE #7662: page request: krita
It seems that [krita manual page](https://manned.org/krita) here is outdated compared to `krita --help` output. I've notified its author about this problem.

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- closes #7662

- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** 5.0.0

### CODE REVIEW

File: pages/linux/krita.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+`krita`
+
+- Start without the splash screen:
+
+`krita --nosplash`
+
+- Open the specified files:
+
+`krita {{path/to/image_file1.ext path/to/image_file2.ext ...}}`
```
Comment (navarroaxel) [inline pages/linux/krita.md:17]:
```suggestion
`krita {{path/to/image_file1 path/to/image_file2 ...}}`
```

Comment (EmilySeville7cfg) [reply pages/linux/krita.md:17]: Without extensions `krita` can't open files. That's why I use `.ext` to underline that extension is mandatory.

Comment (navarroaxel) [reply pages/linux/krita.md:17]: The `path/to/file` is a placeholder for any kind of filename, with extension or not. Krita doesn't need an explicit extension to recognize the file.

Comment (EmilySeville7cfg) [reply pages/linux/krita.md:17]: Just tested. Yes, you are right, without extensions everything works. ;)

**Review by navarroaxel — APPROVED** [review]

**Review by pixelcmtd — APPROVED** [review]

**Pull request merged**